### PR TITLE
Feedback cubit

### LINKED ISSUE #44: ContactUs Screen FeedbackForm -> Cubit
### Is your feature request related to a problem? Please describe.

We need to design the cubit for FeedbackForm on the ContactUs Screen.

### Describe the solution you'd like

1. should have states: Initial, Loading, success, error.
2. fake api repository.
3. tests.

### PR DESCRIPTION
**Pull request by Hard-Coder05**
### Description

Made the feedback cubit.

**Type of Change:** (Delete irrelevant options)
- Code

**Code/Quality Assurance Only** (Delete irrelevant options)

- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?

Tested locally.

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] Any dependent changes have been merged
- [ ] I have added one/more required dependencies
- [x] I have formatted the code using dart format (dartfmt)

**Code/Quality Assurance**
- [x] My changes generate no new warnings 
- [ ] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by werainkhatri** [comment]
Its failing ci checks.

**Comment by Hard-Coder05** [comment]
> Its failing ci checks.

Its working now!

File: lib/screens/home/cubit/feedback_state.dart [hunk 1]
```diff
@@ -0,0 +1,27 @@
+part of 'feedback_cubit.dart';
+
+@immutable
```
Comment (werainkhatri) [inline lib/screens/home/cubit/feedback_state.dart]: reason for calling it `@immutable`?

**Review by werainkhatri — CHANGES_REQUESTED** [review]
LGTM

File: lib/screens/home/cubit/feedback_state.dart (hunk 1 above)
Comment (Hard-Coder05) [reply lib/screens/home/cubit/feedback_state.dart]: Nothing

**Comment by Hard-Coder05** [comment]
@werainkhatri The PR is ready to get merged!

**Review by werainkhatri — APPROVED** [review]

**Pull request merged**